### PR TITLE
Updates to pDAO protocol parameters table

### PR DESCRIPTION
**Pull request by sckuzzle**

### CODE REVIEW

File: RPIPs/RPIP-46.md [hunk 1]
```diff
@@ -31,13 +31,15 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
...
-| Name                                               | Type       | Initial Value |
-|----------------------------------------------------|------------|---------------|
-| `node_operator_commission_share`                   | pct        | `5`           |
-| `node_operator_commission_share_council_adder`     | pct        | `0`           |
-| `voter_share`                                      | pct        | `9`           |
-| `max_node_operator_commission_share_council_adder` | pct        | `1`           |
-| `allowlisted_controllers`                          | address [] | `[]`          |
+| Name                                               | Type       | Initial Value | Guard Rails                                                                                   |
+|----------------------------------------------------|------------|---------------|-----------------------------------------------------------------------------------------------|
+| `node_operator_commission_share`                   | pct        | `5`           | reth_commission <= 1                                                                          |
```
Comment (orangesamus) [inline RPIPs/RPIP-46.md:36]:
```suggestion
| `node_operator_commission_share`                   | pct        | `5`           | reth_commission <= 100                                                                          |
```

File: RPIPs/RPIP-46.md [hunk 2]
```diff
@@ -31,13 +31,15 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
...
+| `node_operator_commission_share_council_adder`*    | pct        | `0`           | <= `max_node_operator_commission_share_council_adder`; <= `voter_share`; reth_commission <= 1 |
```
Comment (orangesamus) [inline RPIPs/RPIP-46.md:37]:
```suggestion
| `node_operator_commission_share_council_adder`*    | pct        | `0`           | <= `max_node_operator_commission_share_council_adder`; <= `voter_share`; reth_commission <= 100 |
```

**Comment by orangesamus** [comment]
Should have upper limit of 100% not 1% right?
Everything else looks good to me

**Comment by sckuzzle** [comment]
This is not ready for merge - I've reverted some previously merged changes on accident.  Will have time to sort out git stuff in a few hours.

**Comment by orangesamus** [comment]
:+1:

File: RPIPs/RPIP-46.md [hunk 3]
```diff
@@ -31,13 +31,15 @@ This RPIP is part of a set of proposals motivated by a desire to rework Rocket P
...
-| `voter_share`                                      | pct        | `9`           |
-| `max_node_operator_commission_share_council_adder` | pct        | `1`           |
-| `allowlisted_controllers`                          | address [] | `[]`          |
+| Name                                               | Type       | Initial Value | Guard Rails                                                                                      |
+|----------------------------------------------------|------------|---------------|--------------------------------------------------------------------------------------------------|
+| `node_operator_commission_share`                   | pct        | `5`           | reth_commission <= 100%                                                                          |
+| `node_operator_commission_share_council_adder`*    | pct        | `0`           | <= `max_node_operator_commission_share_council_adder`; <= `voter_share`; reth_commission <= 100% |
+| `voter_share`                                      | pct        | `9`           | reth_commission <= 100%                                                                          |
```
Comment (Valdorff) [inline RPIPs/RPIP-46.md:38]: should the reth_commission rails be `0% <= reth_commission <= 100%`?

Comment (Valdorff) [reply RPIPs/RPIP-46.md:38]: RP contracts only use unsigned numbers, so this isn't seen as an operative difference

**Pull request merged**